perfmetrics/scripts/testing_on_gke/examples/fio/loading-test/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 The Kubernetes Authors.
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default values for unet3d-loading-test.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: ubuntu:24.04
+bucketName: gke-dlio-test-data
+scenario: local-ssd
+nodeType: n2-standard-96
+
+resourceLimits:
+  cpu: 100
+  memory: 400Gi
+
+fio:
+  readType: read
+  fileSize: 64K
+  blockSize: 64K
+  filesPerThread: "20000"
+
+gcsfuse:
+  metadataCacheTTLSeconds: "6048000"
+  metadataStatCacheCapacity: "-1"
+  metadataTypeCacheCapacity: "-1"
+  fileCacheCapacity: "-1"
+  fileCacheForRangeRead: "true"
```

Comment:
Add an empty line at the end of the file.

Reply:
done